src/mud/examples/adcirc.py

```diff
@@ -218,22 +222,24 @@ def adcirc_time_window(
         pca_vector_plot(
             adcirc_prob, t_mask, msize=msize, max_plot=max_plot, title=title
         )
-        save_figure(
-            f"pca_vecs_{num_components}_{ndata}",
-            save_path,
-            close_fig=close_fig,
-            dpi=dpi,
-            bbox_inches="tight",
-        )
+        if save_path:
```

Comment:
Should be 
`if save_path is not None:`
See [PEP 8](https://peps.python.org/pep-0008/)